apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/txtresolver/PropertyResolver.java

```diff
@@ -155,7 +155,7 @@ private boolean isBlankItem(ItemDTO item) {
   }
 
   private boolean isBlankItem(String line) {
-    return "".equals(line);
+    return  "".equals(line.trim());
```

Comment:
how about `Strings.nullToEmpty(line).trim().isEmpty()` which works even `line` is null?

Reply:
I've thought about this, but seems `line` cannot be null in this case, anyway, it's a coding defection here, I agree with you